### PR TITLE
[Sketcher] Redo Improve angle & radius constraint rendering (#11507)

### PR DESCRIPTION
**Pull request by FlachyJoe**
Two commits of #11507 were lost, certainly when solving conflicts for #9166.

### CODE REVIEW

File: src/Mod/Sketcher/Gui/EditModeConstraintCoinManager.cpp [hunk 1]
```diff
@@ -1311,27 +1330,42 @@ void EditModeConstraintCoinManager::processConstraints(const GeoListFacade& geol
...
+                                    helperStartAngle = angle;
+                                    helperRange = startAngle - angle;
+                                }
+                                else {
+                                    if (angle < endAngle) {
+                                        angle += 2 * M_PI;
+                                    }
+                                    helperStartAngle = endAngle;
+                                    helperRange = angle - endAngle;
+                                }
```
Comment (benj5378) [inline src/Mod/Sketcher/Gui/EditModeConstraintCoinManager.cpp:1361]: Would it be suitable to move this its own function?

Comment (PaddleStroke) [reply src/Mod/Sketcher/Gui/EditModeConstraintCoinManager.cpp:1361]: Perhaps, though this PR is about restoring what was removed by error. So I'm not sure refactoring belongs to this PR.

**Comment by chennes** [comment]
Thanks for hunting this down.

**Comment by maxwxyz** [comment]
@FlachyJoe @PaddleStroke not sure if these were present before but the helpers for angles and distances on arcs are not present:
grafik

**Comment by PaddleStroke** [comment]
No they were not implemented yet.
Only line-line angle and radius has helper implemented

**Pull request merged**